documentation/src/main/resources/pages/ditto/httpapi-protocol-bindings-cloudevents.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Cloud Events HTTP protocol binding
+keywords: binding, protocol, http, cloudevents
+tags: [binding, protocol, http]
+permalink: httpapi-protocol-bindings-cloudevents.html
+---
+
+Implements the [HTTP Protocol Binding for CloudEvents - Version 1.0](https://github.com/cloudevents/spec/blob/v1.0/http-protocol-binding.md).
+
+Unless mentioned otherwise, the endpoint following the Cloud Events specification for the HTTP binding in version 1.0.
+
+## Cloud Events features
+
+The Cloud Events endpoint provides an alternative to the other connectivity APIs to stream data into your instance.
+
+## Cloud Events endpoint
+
+The Cloud Events endpoint is accessible at these URLs (depending on which API version to use):
+
+```
+http://localhost:8080/api/1/cloudevents
+http://localhost:8080/api/2/cloudevents
+```
+
+### Authentication
+
+A user who connects to the Cloud Events endpoint can be authenticated by using
+
+* HTTP BASIC Authentication by providing a username and the password of a user managed within nginx or
+* a JSON Web Token (JWT) issued by an OpenID connect provider.
+
+See [Authenticate](basic-auth.html) for more details.
+
+## Cloud Events protocol format
+
+The source must be a Cloud Event, encoded in the format for the HTTP binding. It may be encoded in *binary content mode*
+or in *structural content mode*.
+
+The *data type* of the event must be `application/json`.
+
+The *data schema* must start with `ditto:`, for example `ditto:org.eclipse.vorta:TestDevice:1.0.0`.
```

Comment:
If I understood the CloudEvent spec correctly, this is the `dataschema`, not the `data schema`.
I would like to omit the reference to Vorto here as this information (the Vorto model) can currently not be used to do anything meaningful.

Thinking about it: do we even need to specify this? It is optional in the spec and we don't do anything meaningful with that ..
So adding it to a cloud-event brings no value.